haproxy/tests/conftest.py

```diff
@@ -60,44 +59,27 @@ def wait_for_haproxy():
 
 @pytest.fixture(scope="session")
 def spin_up_haproxy():
```

Comment:
While you're at it, can we change the name to `haproxy_container` or something ? A declarative name is better suited for a fixture than a imperative one.